### DISCUSSION TITLE
Featurebox plugin

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5283] [General]
Hi,

I want to use the featurebox plugin on my website. The images are cropped, but I want to display the whole image instead of a cropped image. I think I nearrowed it down to the featurebox_template.php (see screenshot attached).

I tried to change 'crop=1' to 'crop=0'. Also I tried to  delete the 'crop=1' from the rows. Leaving out '{SETIMAGE: w=600&h=450&crop=1}' makes it better, but it seems that it is still not the whole image.

Can someone help me with this to display the whole image?

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/e107inc/e107/assets/140949730/7c8610a4-fddf-4b1a-8a80-f627037c630c)

**Comment by Jimmi08** [comment]
Try to use 
`{SETIMAGE: w=0&h=0}  `
- it should render full image

**Reply by PH5HP** [reply]
Hi Jimmi08,

Still not the whole image displayed. Are there more variables which determine the size of the image displayed? The result with your solution is the same as if I left out the SETIMAGE part. I think somewhere else the size is determined. No idea where.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by Jimmi08** [reply]
In gallery, default image sizes. But it can be anything, theme settings, core issue, overridden with other plugin...   For sure w=0 and h=0  should display full image. Or it can be aw=0&ah=0 ? 

Maybe this is related:
https://github.com/e107inc/e107/issues/1010
https://github.com/e107inc/e107/issues/1631

Maybe this helps:
https://github.com/e107inc/e107/issues/2319

**Reply by PH5HP** [reply]
Hi Jimmi08,

Thank you for helping me out. 

I think it is the image I am using. Just tried another image and it seems fine with '{SETIMAGE: w=0&h=0}' .

 Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by PH5HP** [reply]
Hi,

I have it working now.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/e107inc/e107/assets/140949730/2d72d46b-e6ec-4ab9-a58a-4b7524b2af09)
